docs/applications/big-data/how-to-move-machine-learning-model-to-production.md

```diff
@@ -74,73 +74,74 @@ Keras is a deep learning library for Python. It provides an object-oriented inte
-    :   ~~~
-        from keras.models import Sequential
-        from keras.layers import Dense, Dropout, Activation, Flatten
-        from keras.layers import Convolution2D, MaxPooling2D
-        from keras.utils import np_utils
-        from keras.datasets import mnist
-
-        (X_train, y_train), (X_test, y_test) = mnist.load_data()
-        X_train = X_train.reshape(X_train.shape[0],1,28,28)
-        X_test  = X_test.reshape(X_test.shape[0],1,28,28)
-        X_train = X_train.astype('float32')
-        X_test = X_test.astype('float32')
-        X_train /= 255
-        X_test /= 255
-
-        Y_train = np_utils.to_categorical(y_train, 10)
-        Y_test = np_utils.to_categorical(y_test, 10)
-
-        model = Sequential()
-
-        model.add(Convolution2D(32,(3,3),activation='relu',input_shape=(1,28,28),dim_ordering='th'))
-        model.add(MaxPooling2D(pool_size=(2,2)))
-        model.add(Dropout(0.25))
-        model.add(Flatten())
-        model.add(Dense(128, activation='relu'))
-        model.add(Dropout(0.5))
-        model.add(Dense(10, activation='softmax'))
-        model.compile(loss='categorical_crossentropy',
-                      optimizer='adam',
-                      metrics=['accuracy'])
-
-        model.fit(X_train, Y_train,
-                  batch_size=32, nb_epoch=5, verbose=1)
-
-        model.save('my_model.h5')
-        ~~~
+    {{< file "~/models/mnist_model.py" >}}
+from keras.models import Sequential
+from keras.layers import Dense, Dropout, Activation, Flatten
+from keras.layers import Convolution2D, MaxPooling2D
+from keras.utils import np_utils
+from keras.datasets import mnist
+
+(X_train, y_train), (X_test, y_test) = mnist.load_data()
+X_train = X_train.reshape(X_train.shape[0],1,28,28)
+X_test  = X_test.reshape(X_test.shape[0],1,28,28)
+X_train = X_train.astype('float32')
+X_test = X_test.astype('float32')
+X_train /= 255
+X_test /= 255
+
+Y_train = np_utils.to_categorical(y_train, 10)
+Y_test = np_utils.to_categorical(y_test, 10)
+
+model = Sequential()
+
+model.add(Convolution2D(32,(3,3),activation='relu',input_shape=(1,28,28),dim_ordering='th'))
+model.add(MaxPooling2D(pool_size=(2,2)))
+model.add(Dropout(0.25))
+model.add(Flatten())
+model.add(Dense(128, activation='relu'))
+model.add(Dropout(0.5))
+model.add(Dense(10, activation='softmax'))
+model.compile(loss='categorical_crossentropy',
+              optimizer='adam',
+              metrics=['accuracy'])
+
+model.fit(X_train, Y_train,
+          batch_size=32, nb_epoch=5, verbose=1)
+
+model.save('my_model.h5')
+
+{{< /file >}}
+
 
 3.  Run the script:
 
         python ./mnist_model.py
 
     There may be a warning message as shown below from a pip or conda installation which means installing from source could offer superior performance.
 
-    {:.output}
-    ~~~
+    {{< output >}}
+~~~
     The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
     ~~~
```

Comment:
Weird squiggly clean-up

Reply:
@EdwardAngert Can you fix?

Reply:
find squiggles, remove squiggles, realign

```
    {{< output >}}
stuff to output aligns left
then the closing /output does too
{{< /output>
```